### ISSUE TITLE
Invalid position given by 'W119'

### ISSUE THREAD

**Issue by sukso96100** [AtomLinter/linter-jshint#442]
JSHint returned a point that did not exist in the document being edited.
Rule: `W119`
Requested start point: 35:33

Debug information:
```
Atom version: 1.18.0
linter-jshint version: v3.1.4
JSHint version: jshint v2.9.5
Hours since last Atom restart: 13.9
Platform: darwin
Current file's scopes: [
  "source.js"
]
linter-jshint configuration: {
  "executablePath": "",
  "lintInlineJavaScript": false,
  "disableWhenNoJshintrcFileInPath": false,
  "scopes": [
    "source.js",
    "source.js-semantic"
  ],
  "jshintFileName": ".jshintrc",
  "jshintignoreFilename": ".jshintignore"
}
```

**Comment by Arcanemagus** [comment]
Is your file indented with tabs?

**Comment by sukso96100** [comment]
yes

**Comment by Arcanemagus** [comment]
Okay, you should follow https://github.com/jshint/jshint/issues/3151 for updates on when this will be fixed then, there is currently a major bug in JSHint where most rules report incorrect positions for files indented using tabs.

For anyone directed here and isn't using tabs to indent your file please file a new issue with a file to reproduce it on the JSHint repo, or file one here and I'll try to help you in getting one filed there where it can be fixed.

**Comment by Arcanemagus** [comment]
There are reports that if you set your `indent` property of your JSHint configuration to `1` this issue can be worked around. If you don't have a `.jshintrc` file the following should work:
```json
{
  "indent": 1
}
```
